test/unit/ObjectMakrTests/processFunctions.js

```diff
@@ -0,0 +1,35 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("processes the correct number of functions", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            var inheritance = mocks.mockInheritance();
+
+            // Act
+            objectMaker.functions = {};
```

Comment:
it would work if I changed 3 to whatever the actual value is...I believe it would be 10

(thought I hit comment on this, woops)